laborator/content/etapele-compilarii/06-obj-link-dev/Makefile

```
@@ -1,2 +1,14 @@
-PROGNAME := main
-include ../../utils/Makefile.generic
+CC = gcc
+CFLAGS = -g -Wall -Wextra -Werror -fno-pic -masm=intel
+
+.PHONY: all clean
+
+all: main
+
+# ** TO DO **
+# Add dependecies here so that the Makefile works
+main: main.c
+	$(CC) $(CFLAGS) -o $@ $^
+
+clean:
+	-rm -f main main.o	
```

```suggestion
	-rm -f main main.o
```
`checkpatch.pl` was also complaining about this. Always check out these errors when you create PRs. To silence old ones, squash your commits into a single one and then `git push -f`.